ci/install-package-dependencies.sh

```diff
@@ -86,6 +86,22 @@ echo "$EDATOOLS_REPO" > "$TMPDIR/obs.list"
 sudo mv "$TMPDIR/obs.asc"  /etc/apt/trusted.gpg.d/obs.asc
 sudo mv "$TMPDIR/obs.list" /etc/apt/sources.list.d/edatools.list
 
+# Install gcc-9 and set it as the default
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+sudo $APT_CMD update
+sudo $APT_CMD install -y gcc-9 g++-9
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 90
+sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 90
+
+# Install bazel from bazel's apt repository
+sudo $APT_CMD install -y apt-transport-https gnupg
+curl -fsSL https://bazel.build/bazel-release.pub.gpg \
+  | gpg --dearmor > bazel.gpg
+sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
+echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" \
+  | sudo tee /etc/apt/sources.list.d/bazel.list
```

Comment:
Nit: Could we add a `>/dev/null` to the end of this command? Otherwise I think we're going to echo the line to the terminal each time.